calico.go

```diff
@@ -56,8 +54,16 @@ func cmdAdd(args *skel.CmdArgs) error {
 
 	utils.ConfigureLogging(conf.LogLevel)
 
+	if conf.RequireNodenameFile {
+		// Configured to use the nodename file - don't start until it exists.
+		if _, err := os.Stat("/var/lib/calico/nodename"); err != nil {
+			return fmt.Errorf("error reading /var/lib/calico/nodename: %s: is calico/node running?", err)
+		}
+		logrus.Debug("/var/lib/calico/nodename exists")
+	}
+
 	// Allow the nodename to be overridden by the network config
```

Comment:
Now that nodename is being instantiated here, I don't think this comment is right

Reply:
I don't think the comment was ever right.